### ISSUE TITLE
I have problem

### ISSUE THREAD

**Issue by lgs777** [Megalan/PatreonDownloader#6]
How can I solve this problem? plz help..

2020-04-24 20:43:32.9199 FATAL Fatal error, application will be closed: PatreonDownloader.Engine.Exceptions.PatreonDownloaderException: Unable to retrieve campaign id
   at PatreonDownloader.Engine.PatreonDownloader.Download(String creatorName, PatreonDownloaderSettings settings) in E:\Sources\BigProjects\PatreonDownloader\PatreonDownloader.Engine\PatreonDownloader.cs:line 162
   at PatreonDownloader.App.Program.RunPatreonDownloader(String creatorName, Boolean headlessBrowser, PatreonDownloaderSettings settings) in E:\Sources\BigProjects\PatreonDownloader\PatreonDownloader.App\Program.cs:line 136
   at PatreonDownloader.App.Program.Main(String[] args) in E:\Sources\BigProjects\PatreonDownloader\PatreonDownloader.App\Program.cs:line 58

**Comment by Megalan** [comment]
1) Are you sure you are using latest release?
2) Can you post creator's name? You can send it to my email if you want to keep it private.

**Comment by lgs777** [comment]
> 1. Are you sure you are using latest release?
> 2. Can you post creator's name? You can send it to my email if you want to keep it private.


Thx for reply.
1. Yes I am using 0.91
2. https://www.patreon.com/vincentschwenk/

**Comment by Megalan** [comment]
Unfortunately I cannot reproduce this on my end.

I have made a test build which dumps the page used to retrieve creator id into the file called **creatorpagedump.html**. It would be great if you could download this build and send me this file. 

https://mega.nz/file/y1N2iaaQ#NVQtoLUmCnqa3aZp5MWjnOWWn-oeS9JXMcNst6GOOtA

**Please note:** this file contains your account information such as email, first name, last name, linked accounts, subscriptions list. I advise you to open it in any text editor and manually search and edit some of that information and use email to send it to me.

You can safely remove everything starting with the line:
`"currentUser": {`

and ending with the line:
`"self": "https://www.patreon.com/api/user/`

**Comment by Megalan** [comment]
Issue closed as no further information was provided.

**Comment by lgs777** [comment]
Sorry! I wasn't familiar with using GITHUB, so I was late to check this article. I thought this was deleted!!

[creatorpagedump.zip](https://github.com/Megalan/PatreonDownloader/files/4621853/creatorpagedump.zip)

**Comment by Megalan** [comment]
I have checked the creator name from the other issue. It appears that you have a typo in creator name.

You are using vi**nect**schwenk while the proper one is vi**ncent**schwenk

**Comment by lgs777** [comment]
Sorry to bother you. I tried copying the creator's name by dragging. I tried it again and again, but the symptoms are the same.
I succeeded only once in my first attempt after formatting the computer. But after that I get an error. The symptoms are the same if you try it with another computer in the house. What is the problem?

**Comment by Megalan** [comment]
Unfortunately I don't know how to help you. The fact that this problem was not reported by other users tells me that you're doing something wrong and I only provide support for cases when the problem is in the software itself.

All I can say is that it works fine on my side using the following command line on windows 10 version 1909 x64:
> .\PatreonDownloader.App.exe --creator vincentschwenk --descriptions --embeds --json --campaign-images --verbose --no-headless

I close this issue and mark it as wontfix.

**Comment by lgs777** [comment]
Thank you for your efforts. I will solve it. I will post here once I know why. Thank you.
